app/controllers/foreman_tasks/api/tasks_controller.rb

```diff
@@ -127,7 +211,7 @@ def active_scope(scope, search_params)
       end
 
       def action_types_scope(scope, search_params)
-        if action_types = search_params[:action_types]
+        if action_types == search_params[:action_types]
```

Comment:
@iNecas - i thought this was a simple typo but did you intend it to be an assignment inside an if statement?